Workaround for binutils 2.17 linking things on mips in a way musl doesn't like.

diff --git a/arch/mips/crt_arch.h b/arch/mips/crt_arch.h
index 21e139b..f4ba02a 100644
--- a/arch/mips/crt_arch.h
+++ b/arch/mips/crt_arch.h
@@ -4,13 +4,16 @@ __asm__(
 ".text \n"
 ".global _" START "\n"
 ".global " START "\n"
+".global " START "_data\n"
 ".type   _" START ", @function\n"
 ".type   " START ", @function\n"
+".type   " START "_data, @function\n"
 "_" START ":\n"
 "" START ":\n"
 "	bal 1f \n"
 "	 move $fp, $0 \n"
-"2:	.gpword 2b \n"
+"" START "_data: \n"
+"	.gpword " START "_data \n"
 "	.gpword " START "_c \n"
 ".weak _DYNAMIC \n"
 ".hidden _DYNAMIC \n"
